code/modules/mob/living/simple_animal/abnormality/he/red_shoes.dm

```diff
@@ -0,0 +1,448 @@
+	SEND_GLOBAL_SIGNAL(COMSIG_GLOB_ABNORMALITY_BREACH, src)
+	FearEffect()
+
+/mob/living/simple_animal/hostile/abnormality/red_shoes/Found(atom/A)//The solo breach generally sticks together
+	if(istype(A, /mob/living/simple_animal/hostile/red_shoe))
+		var/mob/living/simple_animal/hostile/red_shoe/S = A
+		if(S.stat != DEAD && !S.target && !S.client && faction_check_mob(S))//cannibalized from steel ordeals
+			S.Goto(src,S.move_to_delay,1)
+
+/mob/living/simple_animal/hostile/abnormality/red_shoes/AttackingTarget()
+	. = ..()
+	if(!ishuman(target))
+		return
+	var/mob/living/carbon/human/H = target
+	if(H.stat >= SOFT_CRIT || H.health < 0)
+		ChopFeet(H)
+
+/mob/living/simple_animal/hostile/abnormality/red_shoes/proc/ChopFeet(mob/living/carbon/human/H)
+	var/obj/item/bodypart/l_foot = H.get_bodypart(BODY_ZONE_L_LEG)//Feet are defined asBODY_ZONE_PRECISE_L_FOOT. Does the dismember proc not affect them?
+	var/obj/item/bodypart/r_foot = H.get_bodypart(BODY_ZONE_R_LEG)
+	if(HAS_TRAIT(H, TRAIT_NODISMEMBER))
+		return
+	playsound(src, 'sound/abnormalities/redshoes/RedShoes_Kill.ogg', 100, 1)
+	var/severance = (l_foot?.dismember() && r_foot?.dismember())
+	if(!severance)
+		return
+	H.adjustBruteLoss(500)//kill the target
+
+//Simple mob
+/mob/living/simple_animal/hostile/red_shoe
+	name = "Red Shoe"
+	desc = "Teeth and leg bones jut out of this ragged shoe, as if the wearer's will was made manifest."
+	health = 800
+	maxHealth = 800
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "redshoes_breach2"
+	icon_living = "redshoes_breach2"
+	icon_dead = "redshoes_breach2"//dels on death
+	gender = NEUTER
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.5, WHITE_DAMAGE = 1, BLACK_DAMAGE = 1.5, PALE_DAMAGE = 1.5)
+	del_on_death = TRUE
+	deathmessage = "crumples into a pile of bones."
+	attack_sound = 'sound/abnormalities/redshoes/RedShoes_Attack.ogg'
+	melee_damage_lower = 15
+	melee_damage_upper = 30
+	melee_damage_type = RED_DAMAGE
+	speed = 1
+	move_to_delay = 3
+	var/steppy = 0
+
+/mob/living/simple_animal/hostile/red_shoe/Moved()
+	. = ..()
+	if(steppy == 2)
+		playsound(get_turf(src), 'sound/abnormalities/redshoes/step1.ogg', 30, 0, 3)
+		steppy = 0
+		return
+	++steppy
+
+/mob/living/simple_animal/hostile/red_shoe/AttackingTarget()
+	. = ..()
+	if(!ishuman(target))
+		return
+	var/mob/living/carbon/human/H = target
+	if(H.stat >= SOFT_CRIT || H.health < 0)
+		ChopFeet(H)
+
+/mob/living/simple_animal/hostile/red_shoe/proc/ChopFeet(mob/living/carbon/human/H)
+	var/obj/item/bodypart/l_foot = H.get_bodypart(BODY_ZONE_L_LEG)//Feet are defined asBODY_ZONE_PRECISE_L_FOOT. Does the dismember proc not affect them?
+	var/obj/item/bodypart/r_foot = H.get_bodypart(BODY_ZONE_R_LEG)
+	if(HAS_TRAIT(H, TRAIT_NODISMEMBER))
+		return
+	playsound(src, 'sound/abnormalities/redshoes/RedShoes_Kill.ogg', 100, 1)
+	var/severance = (l_foot?.dismember() && r_foot?.dismember())
+	if(!severance)
+		return
+	H.adjustBruteLoss(500)//kill the target
```

Comment:
Previous applies here as well.

Reply:
Dismemberment occurs when you're already in critical condition. It is meant to be a finisher.